src/Sylius/Behat/Context/Ui/Admin/ManagingCustomersContext.php

```diff
@@ -15,6 +15,7 @@
 
 use Behat\Behat\Context\Context;
 use Sylius\Behat\Page\Admin\Crud\IndexPageInterface;
+use Sylius\Behat\Page\Admin\Customer as Customer;
```

Comment:
Something went wrong with this refactor

Reply:
Yes, I think we can use CustomerIndexPageInterface on constructor arguments..